uwblueprint/ccbc
# able to create user with rand password and send login details via email

## Notion ticket link

[Admin Authentication: Create Account] (https://www.notion.so/uwblueprintexecs/Admin-Authentication-Create-Account-b2b23688973543458dc621ada42fc469)



## Implementation description
* made a new function called sendPasswordSetupLink() that sends an email to the user with their temp login details. Decided to make a new function as the name was more suitable and the functionality was tailored to giving temp details and having a link to set up password. Also, this should not affect changes elsewhere as it is a new function.



## Steps to test
1. Create a new user from the front end with first name, email, password
2. Verify that user account was created in firebase and sql database
3. verify that you got an email with your login details and a random password 



## What should reviewers focus on?
* should only be minor changes, focus mostly on the new function sendPasswordSetupLink() and the ./register route. 


## Checklist
